x-pack/plugins/siem/public/components/open_timeline/index.test.tsx

```diff
@@ -491,33 +501,6 @@ describe('StatefulOpenTimeline', () => {
     });
   });
 
-  test('it renders the title', async () => {
```

Comment:
Can we check if the tab is re-rendering instead of the title? We should keep our test ;)